addons/mail/models/ir_attachment.py

```diff
@@ -100,11 +103,17 @@ def _to_store(self, store: Store, /, *, fields=None, extra_fields=None):
         )
         for attachment in self:
             data = attachment._read_format(
-                [field for field in fields if field not in ["filename", "size", "thread"]],
+                [field for field in fields if field not in ["filename", "message", "size", "thread"]],
                 load=False,
             )[0]
             if "filename" in fields:
                 data["filename"] = attachment.name
+            if "message" in fields:
+                data["message"] = (
```

Comment:
It's annoying that this is a non-matching field name, as I'm trying to clean them all.
Maybe we should simply return all `message_ids` and do the processing in client code.